easybuild/easyconfigs/d/depad-utils/depad-utils-v16.10.2.eb

```diff
@@ -0,0 +1,49 @@
+name = 'depad-utils'
+version = 'v16.10.2'
+
+homepage = 'https://github.com/molgenis/depad-utils/'
+description = """Utilities for deploy admins to manage HPC environments."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'CmdCp'
+
+#
+# Example URL:
+# https://github.com/molgenis/depad-utils/archive/v15.06.1.tar.gz
+#
+source_urls = [('http://github.com/molgenis/%s/archive/' % (name))]
+sources = [('%s.tar.gz' % (version))]
+
+#
+# Install only 'apps' for relevant environment (hpc-2.x) from this repo with EB.
+#
+files_to_copy = [
+    'hpc-2.x/README.md',
+    (['hpc-2.x/apps/software/depad-utils/*'], 'bin/')
+]
+
+#
+# We don't really need a command to install depad-utils, 
+# but as we have to have one in order to reycle the CmdCp easyblock... just ls.
+#
+cmds_map = [('.*', "ls -ahl %(source)s")]
```

Comment:
The tarball EasyBlock does not have the option to list which files you want to copy with "files_to_copy". Tarball will copy everything and that is way to much in this case. Writing a custom EasyBlock would be an option, but then we need to maintain a custom EasyBlock repo in addition to one for EasyConfigs and I found that too much overhead to remove the silly hack with a useless ls.